PermissionsService/Classes/Calendar.swift

```diff
@@ -8,17 +8,15 @@
 import UIKit
 import EventKit
 
-public final class CalendarPermission: PermissionService {
-    
-  public init() {
-    
-  }
+public final class Calendar: PermissionService {
   
-	public var entityType = EKEntityType.event
+  public var entityType = EKEntityType.event
```

Comment:
become 'private let' after Creation two separated classes for Calendar-Event and Reminder permissions